src/views/utility/preprocessing.js

```diff
@@ -149,6 +149,7 @@ export function tokenizeWords(
     return tokenizeChineseSanskrit(
       inputData,
       colorValues,
+      lang,
```

Comment:
This function is basically the same for the tokanization of all languages ... best to keep it the same for all. It looks like this needs refactoring to simplify it a bit.
Also a few superfluous console.log and comments.